### ISSUE TITLE
Assignment 06 Status

### ISSUE THREAD

**Issue by jeehoonkang** [snu-sf/pl2015#69]
Hi all,
- Assignment 06 is issued.
- Please read [Homework.md](Homework.md#how-to-submit) to know how to fetch homework and submit your answer.
  - You have to run `./fetch-homework.sh` to get assignment 06.
  - Do your homework by editing `sf/Assignment06_??.v`.
  - Make sure `make` works without errors.
  - Commit and push to your GitHub repository.
  - Make sure `https://github.com/$YOURID/pl2015` contains the change you made.
- Stay tuned for [README.md](README.md) and this issue on the assignment.
- You are NOT allowed to use the `admit` tactic
   because `admit` simply admits any goal 
   regardless of whether it is provable or not.
  
   But, you can leave `admit` for problems that you cannot prove.
   Then you will get zero points for those problems.
- You are NOT allowed to use the following tactics.
  
   `tauto`, `intuition`, `firstorder`, `omega`.
- Do NOT add any additional `Require Import/Export`.

Sincerely,
Jeehoon

**Comment by jeehoonkang** [comment]
I will upload automated grader for evaluation purposes.

**Comment by Hyunjin95** [comment]
안녕하세요. Assignment06_05의 문제 'Inductive all'이 컴파일이 되지 않아서 찾아보니 중괄호 {P : X -> Prop}이 (P : X -> Prop)이 되어야 하는 데 문제를 잘 못 적으신 것 같습니다. 저와 같은 문제가 있는 분이 계실까봐 올립니다.
There is a wrong usage of a brace in the problem 'Inductive all' in Assignment06_05.v.
I think {P : X -> Prop } should be (P : X -> Prop).

**Comment by jaewooklee93** [comment]
In `Assignment_06_07.v`, proofs for `Example`s are already given in the comments, can we just use it? :)

**Comment by jeehoonkang** [comment]
@jaewooklee93 Yes you can. I believe the essence of the exercise is defining the `nostutter` relation itself. Proving examples with the already-given comments looks good to me.

Jeehoon

**Comment by jeehoonkang** [comment]
I made an automatic grader: http://sf.snu.ac.kr/jeehoon.kang/pl2015/Assignment06_grader.tar.gz
- You have to use it with `bash`, and `sed` should be installed in your machine. You can use these programs in the `martini.snucse.org` and `midori.snucse.org` servers.
- Place your `Assignment06_??.v` files in `submission` folder. NOTE: `original` folder should be intact.
- Run by `./grader.sh`.
- If your submission has a compilation error, then an error message will be present.

Hope this grader will help you prepare a good submission.

Jeehoon

**Comment by rhs0266** [comment]
Thanks for uploading grader!

If someone have problem about using `./grader.sh` such as `~~ permissiong denied`, write `chmod a+x *.sh` at directory which contains `./grader.sh`.

**Comment by jeehoonkang** [comment]
Submissions are collected.

**Comment by jeehoonkang** [comment]
Delay submissions are collected.
